frontend/src/framework/store/store.package.ts

```diff
@@ -44,6 +46,26 @@ type ExtraArguments = {
     storage: typeof storage;
 };
 
+const combinedReducer = combineReducers({
+    auth: authReducer,
+    users: usersReducer,
+    lms: lmsReducer,
+    employerOnBoarding: employerOnboardingReducer,
+    talentOnBoarding: talentOnBoardingReducer,
+    employer: employerReducer,
+    app: appReducer,
+    candidate: candidateReducer,
+});
+
+type RootState = ReturnType<typeof combinedReducer>;
+
+const rootReducer: Reducer = (state: RootState, action: AnyAction) => {
+    if (action.type === 'auth/resetEntireStore') {
```

Comment:
you can listen singOut action

Reply:
actually we can't. if I understand this stackoverflow article and especially the comments below it correctly, our resetStore reducer needs to be empty to pass the logic to the rootReducer and process it there. 
I tried using 'auth/sign-out' but it does not work because there we have our logic to delete token from localStorage.